topsf/tasks/inference_v2/combine_base.py

```diff
@@ -0,0 +1,105 @@
+# coding: utf-8
+
+import luigi
+import law
+import os
+
+from columnflow.tasks.framework.base import Requirements, CommandTask
+from columnflow.tasks.framework.mixins import (
+    CalibratorsMixin, SelectorStepsMixin, ProducersMixin, InferenceModelMixin,
+)
+from columnflow.tasks.framework.remote import RemoteWorkflow
+from topsf.tasks.inference import CreateDatacards
+from columnflow.util import dev_sandbox
+
+from topsf.tasks.inference_v2.fit_mixin import FitMixin
+
+
+class CombineBaseTask(
+    FitMixin,
+    CreateDatacards,
+    InferenceModelMixin,
+    ProducersMixin,
+    SelectorStepsMixin,
+    CalibratorsMixin,
+    law.LocalWorkflow,
+    RemoteWorkflow,
+    CommandTask,
+):
+    sandbox = dev_sandbox(law.config.get("analysis", "combine_sandbox"))
+
+    combine_verbosity = luigi.IntParameter(
```

Comment:
For simplicity, this could just be `--verbosity` and could be forwarded to the underlying command by tasks whenever it is supported.
```suggestion
    verbosity = luigi.IntParameter(
```